app/patient-details/components/patientDetails.js

```diff
@@ -44,6 +41,11 @@
         .catch(function () {
           notifier.error(translateFilter('COMMON_MESSAGE_ERROR_ACTION'));
         });
+
+      configurationService.getAddressLevels()
+        .then(function (addressLevels) {
+          vm.addressLevels = addressLevels;
```

Comment:
In some of your other check ins there was was .routes.js file that resolved vm properties. Will this file eventually have a .routes file and will this eventually move into the resolve object?

Reply:
Yes. It's already used in some `.routes`, but the component is missing `bindings: { patient: '<' }` in its definition.

I still don't know whats the best approach for loading this data [controller-activation-promises](https://github.com/johnpapa/angular-styleguide/tree/master/a1#controller-activation-promises) vs  [route-resolve-promises](https://github.com/johnpapa/angular-styleguide/tree/master/a1#route-resolve-promises).